plugin/editorconfig.vim

```diff
@@ -192,12 +196,16 @@ function! s:GetFilenames(path, filename) " {{{1
     return l:path_list
 endfunction " }}}1
 
-function! s:UseConfigFiles() abort " Apply config to the current buffer {{{1
+function! s:UseConfigFiles(allowEmpty) abort " Apply config to the current buffer {{{1
     let b:editorconfig_tried = 1
     let l:buffer_name = expand('%:p')
-    " ignore buffers without a name
+
     if empty(l:buffer_name)
-        return
+        if g:EditorConfig_enable_for_new_buf
+            let l:buffer_name = getcwd()
```

Comment:
I wonder if this should be something like this?---
```suggestion
            let l:buffer_name = getcwd() . "/."
```
Per [the spec](https://editorconfig-specification.readthedocs.io/#id6), if `getcwd()` returns `/foo/bar`, the core would not look for `/foo/bar/.editorconfig`.  Instead, it would start the search with `/foo/.editorconfig`.

Would you please test this case and update this line if necessary?  Thanks!

Reply:
There doesn't seem to be any problem with just bare path, but since the spec say so, better to be safe than sorry